### ISSUE TITLE
async data error

### ISSUE THREAD

**Issue by fxding** [lordfriend/nya-bootstrap-select#179]
when get list data from remote server, dropdown button title dose not refresh to reflect selected value.

https://github.com/lordfriend/nya-bootstrap-select/blob/4b9d5ebd48594b6c3554b14521151a90a0c6b1ea/src/nya-bs-select.js#L305

**Comment by lordfriend** [comment]
something wrong with your data update, maybe not in a digest.

**Comment by fxding** [comment]
no, data has been displayed in dropdown list, and selected value has been checked in list, but button title is not updated to selected value.

I followed the code, and saw it only update button title when `deepWatch` is set
